File: src/main/java/no/nav/syfo/controller/domain/sykepengesoknad/Utdanning.java
package no.nav.syfo.controller.domain.sykepengesoknad;

import java.io.Serializable;
import java.time.LocalDate;

public class Utdanning implements Serializable {
    public LocalDate utdanningStartdato;
    public boolean erUtdanningFulltidsstudium;

    public Utdanning withUtdanningStartdato(final LocalDate utdanningStartdato) {
        this.utdanningStartdato = utdanningStartdato;
        return this;
    }

    public Utdanning withErUtdanningFulltidsstudium(final boolean erUtdanningFulltidsstudium) {
        this.erUtdanningFulltidsstudium = erUtdanningFulltidsstudium;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/controller/domain/sykepengesoknad/Utdanning.kt
package no.nav.syfo.controller.domain.sykepengesoknad

import java.io.Serializable
import java.time.LocalDate

class Utdanning : Serializable {
    var utdanningStartdato: LocalDate? = null
    var erUtdanningFulltidsstudium = false
    fun withUtdanningStartdato(utdanningStartdato: LocalDate?): Utdanning {
        this.utdanningStartdato = utdanningStartdato
        return this
    }

    fun withErUtdanningFulltidsstudium(erUtdanningFulltidsstudium: Boolean): Utdanning {
        this.erUtdanningFulltidsstudium = erUtdanningFulltidsstudium
        return this
    }
}
